diwako_cbrn.vr/scripts/cbrn/functions/fn_fogPFH.sqf

```diff
@@ -0,0 +1,60 @@
+if (isGamePaused) exitWith {};
+params ["_args", "_idPFH"];
+_args params ["_lastIteration"];
+
+private _player = ace_player;
+
+private _time = cba_missiontime;
+private _delta = _time - _lastIteration;
+_args set [0, _time];
+
+private _uptime = _player getVariable ["cbrn_mask_uptime", 0];
+private _fogging = _player getVariable ["cbrn_mask_fogging", false];
+
+private _fatigue = _player call cbrn_fnc_getFatigue;
+private _fatigueAdditive = 0;
+
+if (cbrn_fogFatigueEnabled) then {
+	_fatigueAdditive = ( _fatigue * cbrn_fogFatigueCoef ) * _delta;
+};
+
+if (_player getVariable["cbrn_mask_on", false]) then {
+
+private _backpack = backpack _player;
```

Comment:
Would suggest to not write into variable. just use directly in the in `cbrn_conditioning` check.

Reply:
Would it make sense to also switch to the ace fnc_hasitem here?

Reply:
Hm, don't think so. You are retrieving if the unit has a backpack and then compare it to a set list of possible allowed backpacks. The hasItem function is for items within inventory containers.

Reply:
Alright, makes sense. Keeping as is.